.github/workflows/Java_CI.yaml

```diff
@@ -1,16 +1,31 @@
 name: Java CI
 
 on:
+  pull_request:
+    branches:
+      - wrims-devops
   push:
-    branches: [ Feature/wrims-core ]
+    branches:
+      - Feature/wrims-core
+      - wrims-devops
 
 jobs:
   build:
 
-    runs-on: adoptopenjdk:8-jre-hotspot
+    runs-on: windows-2022
 
     steps:
     - uses: actions/checkout@v2
 
-    - name: Build with Gradle
-      run: ./gradlew build
\ No newline at end of file
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v2
+      with:
+        java-version: '8'
+        distribution: 'temurin'
+        cache: 'gradle'
+
+    - name: Upload jars
+      uses: actions/upload-artifact@v4.0.0
+      with:
+        name: jars result
+        path: ~/wrims-core/build/libs/*.jar
```

Comment:
I believe the `~` is typoed? Should be a `.` instead.

Reply:
I actually tried both '.' and '~' and in neither case did the build find and upload the jar files. Any suggestions about this?

Reply:
Well I should have noticed this earlier, but you lost your actual gradle build task, so there really are no jars to upload